inference/core/utils/image_utils.py

```diff
@@ -458,7 +461,7 @@ def encode_image_to_jpeg_bytes(image: np.ndarray, jpeg_quality: int = 90) -> byt
     Encode a numpy image to JPEG format in bytes.
 
     Args:
-        image (np.ndarray): The numpy array representing an image.
+        image (np.ndarray): The numpy array representing a BGR image.
```

Comment:
is BGR statement correct here?
maybe we would leave it as is and just @deperecate ?